packages/blade/src/components/Box/BaseBox/storybookArgTypes.ts

```diff
@@ -0,0 +1,239 @@
+import { getOnlyBoxProps } from '../Box';
```

Comment:
This file handles all the argTypes of Box, BaseBox, and styled-props. It tries to auto-generate most props from the actual `getStyledProps` and `getOnlyBoxProps` functions.